### PR TITLE
Fix `lint-swagger` action

### PR DESCRIPTION
**Pull request by silverwind**
- Add detection for swagger changes and run `lint-swagger` on it
- Remove `lint-swagger` from `lint-frontend`
- Remove `lint-md` from `lint-frontend`

### CODE REVIEW

File: .github/workflows/files-changed.yml [hunk 1]
```diff
@@ -63,3 +66,6 @@ jobs:
               - "Dockerfile.rootless"
               - "docker/**"
               - "Makefile"
+
+            swagger:
+              - "templates/swagger/v1_json.tmpl"
```
Comment (silverwind) [inline .github/workflows/files-changed.yml:71]:
```suggestion
              - "templates/swagger/*_json.tmpl"
```

File: .github/workflows/files-changed.yml [hunk 2]
```diff
@@ -63,3 +66,6 @@ jobs:
               - "Dockerfile.rootless"
               - "docker/**"
               - "Makefile"
+
+            swagger:
+              - "templates/swagger/*_json.tmpl"
```
Comment (silverwind) [inline .github/workflows/files-changed.yml]:
```suggestion
              - "templates/swagger/v1_json.tmpl"
```

**Comment by silverwind** [comment]
> Why remove lint-md? Is it covered by another job?

Yes, it's ran through `pull-compliance - docs` already, so does not need to run during frontend, e.g. it ran twice.

**Review by techknowlogick — APPROVED** [review]
Awesome, thanks @silverwind <3

**Review by delvh — APPROVED** [review]

**Pull request merged**